### ISSUE TITLE
Still working with latest feathers?

### ISSUE THREAD

**Issue by 3kynox** [geistinteractive/feathers-filemaker#13]
Hello,

Is it still working with latest feathersJs?

Thanks anyway :)

**Comment by DelfsEngineering** [comment]
No. There are some minor breaking changes. At around v3 of feathers. 
HTH

**Comment by 3kynox** [comment]
Thanks! Any plans to update this?

**Comment by toddgeist** [comment]
I will probably not have time to update it any time soon

**Comment by 3kynox** [comment]
Fair enough to had answered at least. I just hope it won't be forgotten, and possibly if I could help, knowing the parts that needs a rework. I'm working on a project where I'll have to link a VueJs website with a Filemaker on-premises server. My resources currently are here and:
- https://github.com/Luidog/fms-api-client
- https://dbservices.com/articles/filemaker-web-portals-with-vue-js/

But I'm kindly new with Filemaker, not Vue.

Anyway, thanks again :)

**Comment by DelfsEngineering** [comment]
The fm data client linked above can easily be used with feathers. Also there is not too much to use feathers-fm with newer versions. See my PR

**Comment by toddgeist** [comment]
frankly there isn't much to using FileMaker as a REST resource.  It's just that REST.  I pretty much just use it raw with fetch or axios. 

There is some new stuff coming down the line that might make it more interesting.  But for now I just don't see much reason to spend much effort on abstractions
